### PR TITLE
[Doc] Update ByteTrack README for dev-1.x.

### PR DESCRIPTION
**Pull request by pixeli99**

### CODE REVIEW

File: configs/mot/bytetrack/README.md [hunk 1]
```diff
@@ -31,18 +31,73 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT17-half-train | MOT17-half-val |   N    |       -        | 67.7 | 78.6 | 79.2 | 12909 | 21024 |  666  | [config](bytetrack_yolox_x_crowdhuman_mot17-private-half.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500.log.json) |
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT17-half-train |   MOT17-test   |   N    |       -        | 61.7 | 78.1 | 74.8 | 36705 | 85032 | 2049  |   [config](bytetrack_yolox_x_crowdhuman_mot17-private.py)    | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500.log.json) |
```
Comment (GT9505) [inline configs/mot/bytetrack/README.md:37]: keep these unchanged

File: configs/mot/bytetrack/README.md [hunk 2]
```diff
@@ -31,18 +31,73 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT20-train | MOT17-train |   N    |       -        | 57.3 | 64.9 | 71.8 | 33,747 | 83,385 | 1,263 | [config](bytetrack_yolox_x_crowdhuman_mot20-private.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040-9ce38a60.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040.log.json) |
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT20-train | MOT20-test  |   N    |       -        | 61.5 | 77.0 | 75.4 | 33,083 | 84,433 | 1,345 | [config](bytetrack_yolox_x_crowdhuman_mot20-private.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040-9ce38a60.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040.log.json) |
```
Comment (GT9505) [inline configs/mot/bytetrack/README.md:48]: keep these unchanged. 
it seems you change the name of config file

File: configs/mot/bytetrack/README.md [hunk 3]
```diff
@@ -31,18 +31,73 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
+# Example 1: Test motXXhalfval
+# The number after config file represents the number of GPUs used. Here we use 8 GPUs.
+./tools/dist_test.sh \
+    configs/mot/bytetrack/bytetrack_yolox_x_8xb4-80e_crowdhuman-mot17halftrain_test-mot17halfval.py 8 \
+    --checkpoint ./checkpoints/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth
+```
+
+**2.2 Example on MOTxx-test dataset**
+
+If you want to get the results of the [MOT Challenge](https://motchallenge.net/) test set, please use the following command to generate result files that can be used for submission. It will be stored in `./mot_17_test_res`, you can modify the saved path in `test_evaluator` of the config.
```
Comment (GT9505) [inline configs/mot/bytetrack/README.md:79]: forgot to fix the `resfile_path` bug in config

File: configs/mot/bytetrack/README.md [hunk 4]
```diff
@@ -46,3 +46,58 @@ Please note that the MOTA on `MOT20-test` is slightly lower than that reported i
...
 | ByteTrack | YOLOX-X  | CrowdHuman + MOT20-train | MOT20-test  |   N    |       -        | 61.5 | 77.0 | 75.4 | 33,083 | 84,433 | 1,345 | [config](bytetrack_yolox_x_8xb4-amp-80e_crowdhuman-mot20train_test-mot20test.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040-9ce38a60.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040.log.json) |
+
+## Get Started
+
+### 1. Training
+
+Due to the influence of parameters such as learning rate in default configuration file, we recommend using 8 GPUs for training in order to reproduce accuracy. You can use the following command to start the training.
+
+```shell
+# Training Bytetrack on multiple GPUs with following command
```
Comment (GT9505) [inline configs/mot/bytetrack/README.md:57]:
```suggestion
# Training Bytetrack on crowdhuman and mot17-half-train dataset with following command
```

File: configs/mot/bytetrack/README.md [hunk 5]
```diff
@@ -46,3 +46,58 @@ Please note that the MOTA on `MOT20-test` is slightly lower than that reported i
...
+```
+
+If you want to know about more detailed usage of `train.py/dist_train.sh/slurm_train.sh`, please refer to this [document](../../../docs/en/user_guides/4_train_test.md).
+
+### 2. Testing and evaluation
+
+**2.1 Example on MOTxx-halfval dataset**
+
+```shell
+# Example 1: Test motXXhalfval
```
Comment (GT9505) [inline configs/mot/bytetrack/README.md:70]:
```suggestion
# Example 1: Test on motXX-half-val set
```

File: configs/mot/bytetrack/README.md [hunk 6]
```diff
@@ -46,3 +46,58 @@ Please note that the MOTA on `MOT20-test` is slightly lower than that reported i
...
+    configs/mot/bytetrack/bytetrack_yolox_x_8xb4-80e_crowdhuman-mot17halftrain_test-mot17halfval.py 8 \
+    --checkpoint ./checkpoints/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth
+```
+
+**2.2 Example on MOTxx-test dataset**
+
+If you want to get the results of the [MOT Challenge](https://motchallenge.net/) test set, please use the following command to generate result files that can be used for submission. It will be stored in `./mot_17_test_res`, you can modify the saved path in `test_evaluator` of the config.
+
+```shell
+# Example 2: Test motxxtest
```
Comment (GT9505) [inline configs/mot/bytetrack/README.md:82]:
```suggestion
# Example 2: Test on motxx-test set
```

**Review by GT9505 — APPROVED** [review]

File: configs/mot/bytetrack/README.md [hunk 7]
```diff
@@ -46,3 +46,58 @@ Please note that the MOTA on `MOT20-test` is slightly lower than that reported i
...
 | ByteTrack | YOLOX-X  | CrowdHuman + MOT20-train | MOT17-train |   N    |       -        | 57.3 | 64.9 | 71.8 | 33,747 | 83,385 | 1,263 | [config](bytetrack_yolox_x_8xb4-amp-80e_crowdhuman-mot20train_test-mot20test.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040-9ce38a60.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040.log.json) |
 | ByteTrack | YOLOX-X  | CrowdHuman + MOT20-train | MOT20-test  |   N    |       -        | 61.5 | 77.0 | 75.4 | 33,083 | 84,433 | 1,345 | [config](bytetrack_yolox_x_8xb4-amp-80e_crowdhuman-mot20train_test-mot20test.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040-9ce38a60.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040.log.json) |
+
+## Get Started
```
Comment (JingweiZhang12) [inline configs/mot/bytetrack/README.md:50]: `Get started` to keep in line with other titles?

**Review by GT9505 — APPROVED** [review]
